cirrus/server/cirrus/settings.py

```diff
@@ -39,7 +40,7 @@ class MetricsConfiguration:
     channel: str = channel
     data_dir: str = "/var/glean"
     log_level: Union[str, int] = logging.getLevelName("WARNING")
-    max_events_buffer: int = 500
+    max_events_buffer: int = int(glean_max_events_buffer)
```

Comment:
On line 33 you do `cast(int, ...)` so this should already be an int, no?

Reply:
just fixed it